src/lib.rs

```diff
@@ -452,6 +462,9 @@ impl Config {
             grease: true,
             cc_algorithm: CongestionControlAlgorithm::CUBIC,
             hystart: true,
+
+            dgram_recv_max_queue_len: DEFAULT_DGRAM_MAX_QUEUE_LEN,
```

Comment:
Well, actually, there's nothing being allocated, but it the default len is `0` it might make sense to do `VecDeque::with_capacity()` instead of `VecDeque::new()` in `DatagramQueue::new()`.

Reply:
good catch, this used to be conditional on the feature but that changed